### ISSUE TITLE
Neos Backend crashes if newReferenceNodePath points to non-existing node

### ISSUE THREAD

**Issue by lorenzulrich** [psmb/Psmb.FlatNav#40]
If a `newReferenceNodePath` is configured, but the referenced node is not available, the backend crashes without providing a meaningful message.

Message in developer console:

> Error: Please provide either a string, an array or an object containing a `contextPath` to the FlowQuery API.

A speaking error would be nice to have.

**Comment by paavo** [comment]
Hey @lorenzulrich 
Does your problem still exist?

I found a [similar Problem](https://github.com/psmb/Psmb.FlatNav/issues/51#issuecomment-1018331246) when trying to use FlatNav with Content-Nodes (instead of Document-Nodes). 
We try to fix this Issue (with a little help from @dimaip).

Maybe we can disable Node Creation too, if the referenced node in `newReferenceNodePath` is not available.

**Comment by lorenzulrich** [comment]
@paavo Yes, this problem still exists. Thanks for looking into it. I'm interested to learn about a solution for content nodes.

**Comment by lorenzulrich** [comment]
This was fixed in https://github.com/psmb/Psmb.FlatNav/pull/68.
